virtual-host-gatherer/virtual-host-gatherer.spec

```diff
@@ -144,11 +146,17 @@ rm -rf %{buildroot}
 %{_bindir}/%{name}
 %{python_sitelib}/virtual_host_gatherer-*.egg-info
 %{python_sitelib}/gatherer/modules/File.py*
+%{python_sitelib}/gatherer/modules/Azure.py*
+%{python_sitelib}/gatherer/modules/GoogleCE.py*
+%{python_sitelib}/gatherer/modules/AmazonEC2.py*
```

Comment:
Please create a new subpackage for it. If somebody do not have instances in the cloud he could just not install the subpackage and also do not need the dependencies. We have also subpackages for VMware and Kubernetes.

As we use for all Clouds the same backend library, we do not need a subpackage for every single cloud.
One subpackage for all which are supported by libcloud would be good.